index.html

```
@@ -1,1668 +1,670 @@
+    gtag("js", new Date());
+    gtag("config", "G-52KQ8L4SQG");
+    </script>
+</head>
+
+<body>
+<a href="#main-content" class="skip-link">Skip to main content</a>
+
+<!-- TOP NAV (Desktop) -->
+<nav class="navbar" role="navigation" aria-label="Top navigation">
+    <div class="navbar-inner">
+        <span class="navbar-brand" aria-label="Asif Sayyed">> asif.exe</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)